lib/osf-components/addon/components/contributor-list/component.ts

```diff
@@ -1,29 +1,123 @@
 import { tagName } from '@ember-decorators/component';
-import { computed } from '@ember-decorators/object';
+import { action, computed } from '@ember-decorators/object';
 import { alias } from '@ember-decorators/object/computed';
 import { service } from '@ember-decorators/service';
 import Component from '@ember/component';
+import { task } from 'ember-concurrency';
 import DS from 'ember-data';
 import I18N from 'ember-i18n/services/i18n';
+
 import Contributor from 'ember-osf-web/models/contributor';
+import Node from 'ember-osf-web/models/node';
+import { QueryHasManyResult } from 'ember-osf-web/models/osf-model';
+import Ready from 'ember-osf-web/services/ready';
+import defaultTo from 'ember-osf-web/utils/default-to';
+
+import styles from './styles';
 import layout from './template';
 
 export interface Contrib {
     title: string;
     id: string | null;
+    bibliographic?: boolean | null;
 }
 
 @tagName('span')
-export default class ContributorList extends Component {
+export default class ContributorList extends Component.extend({
+    loadContributors: task(function *(this: ContributorList, more?: boolean) {
+        if (!this.node) {
```

Comment:
Rather than turning into a noop if node hasn't been passed in, toss an [`assert`](https://emberjs.com/api/ember/3.4/functions/@ember%2Fdebug/assert) into `didRecieveAttrs` to ensure that the component always has a node.